commit ccdcf6df0f6908f87acc9f3207defc01d4cdc2a0 (refs/changes/98/4098/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-31T15:49:43-07:00 (1 year, 4 months ago)
    
    TRITON-457 sdc-scripts update in TRITON-380 broke cns and sdc-manatee

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 6bfa4a2..deefaef 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
+Subproject commit deefaef587ed3bee2706cb6e53ee3468e682932e
